controllers/rhmi/rhmi_controller.go

```diff
@@ -839,7 +839,7 @@ func (r *RHMIReconciler) preflightChecks(installation *rhmiv1alpha1.RHMI, instal
 		return result, nil
 	}
 
-	if installation.Spec.Type == string(rhmiv1alpha1.InstallationTypeManaged) || installation.Spec.Type == string(rhmiv1alpha1.InstallationTypeManagedApi) {
+	if rhmiv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(installation.Spec.Type)) || rhmiv1alpha1.IsRHMI(rhmiv1alpha1.InstallationType(installation.Spec.Type)) {
```

Comment:
Wonder would it be good to add and use a `isManaged` fucntion for this instead while we're these `isX` functions ? :thinking:

Reply:
Thanks @KevFan, I've addressed your comments, thanks a lot for suggestions :)
I've also added the IsManaged to cover RHMI/RHOAM/MtRHOAM. Thanks!